molecule/test13/converge.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: "Include ansible-role-openhpc"
+      include_role:
+        name: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') | basename }}"
+      vars:
+        openhpc_enable:
+          control: "{{ inventory_hostname in groups['testohpc_control'] }}"
+          batch: "{{ inventory_hostname in groups['testohpc_compute'] }}"
+          runtime: true
+        openhpc_slurm_control_host: "{{ groups['testohpc_control'] | first }}"
+        openhpc_slurm_partitions:
+          - name: "compute"
+        openhpc_cluster_name: testohpc
+        openhpc_slurm_configless: true
+        openhpc_login_only_nodes: 'testohpc_login'
+        openhpc_extra_config:
```

Comment:
Would this work when the key in the config file is reused? Examples are:
```
PartitionName=debug Nodes=tara-c-[001-002] Default=YES MaxTime=02:00:00 DefaultTime=00:30:00 State=UP
PartitionName=standby Nodes=tara-c-[001-006] MaxTime=120:00:00 State=UP
```
and
```
NodeName=tara-c-[001-006] CPUs=4 RealMemory=512 Sockets=2 CoresPerSocket=2 ThreadsPerCore=1 State=UNKNOWN TmpDisk=256
NodeName=tara-m-[001-002] CPUs=8 RealMemory=1024 Sockets=2 CoresPerSocket=4 ThreadsPerCore=1 State=UNKNOWN TmpDisk=512
```
Examples taken from: https://tara-admin.readthedocs.io/en/latest/systems/tara/slurm.html

Maybe we should just say that setting these "multi-valued" options is not supported?

Reply:
Eh. For nrel I specifically did this as a list not a mapping for that very reason (but I forgot it). It make the syntax not nearly as "natural" though:

```yaml
openhpc_extra_config:
  - ['FirstJobId', 13]
  - ['SlurmctldSyslogDebug', 'error']
```

Reply:
@JohnGarbutt , @brtknr any comments on this approach? I'm kind of tempted to go with it as is now (i.e. a mapping) on the basis that really we're trying to let users set simple parameters, not the sort of Partition/Node definitions which are handled by the templating and need to appear multiple times.